tokens.xml

```diff
@@ -8438,6 +8654,173 @@ Sacrifice this creature: This creature deals 1 damage to target creature or play
             <token>1</token>
             <tablerow>2</tablerow>
         </card>
+        <card>
+            <name>Dungeon of the Mad Mage</name>
+            <text>Yawning Portal — You gain 1 life. (→ Dungeon Level)
+Dungeon Level — Scry 1. (→ Goblin Bazaar or Twisted Caverns)
+Goblin Bazaar — Create a Treasure token. (→ Lost Level)
```

Comment:
The treasure token relation is missing.

Reply:
It is already right below the Skeleton relation.